plugins/ocm-backend/src/helpers/kubernetes.ts

```diff
@@ -11,7 +13,7 @@ import { ManagedCluster, ManagedClusterInfo, OcmConfig } from '../types';
 
 export const hubApiClient = (
   clusterConfig: OcmConfig,
-  logger: Logger,
+  logger: Logger | LoggerService,
```

Comment:
the `Logger` type and `LoggerService` are two different ideas, so this should remain as `Logger`. Since these are two distinct types, it will create errors at runtime if we follow this definition. `LoggerService` should somehow give us `Logger` here as happens in the main Backstage backend